webapp/channels/src/components/admin_console/system_user_detail/system_user_detail.tsx

```diff
@@ -398,6 +398,16 @@ export class SystemUserDetail extends PureComponent<Props, State> {
                                         <SheidOutlineIcon/>
                                         <span>{getUserAuthenticationTextField(this.props.intl, this.props.mfaEnabled, this.state.user)}</span>
                                     </label>
+                                    {this.state.user?.auth_data && this.state.user?.auth_service && (
```

Comment:
Is safer to wrap this condition in a Boolean

```suggestion
                                    {Boolean(this.state.user?.auth_data && this.state.user?.auth_service) && (
```

Reply:
Good point, I think that makes the code safer. And the intent more clear